src/components/components/ReinforcementStatement/ReinforcementStatement.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./ReinforcementStatement.scss";
+import ReinforcementImage from "./Assets/reinforcementImage.jpg";
+
+function ReinforcementStatement() {
+  return (
+  <div className="Reinforcement">
+            <div className="Reinforcement__image">
+                <img src='./reinforcementImage.jpg' alt=' Reinforcement Image'/>
+            </div>
+            <div className='Reinforcement__statement'>
+                <h2 className='Reinforcement__header'>
+                    Unleash your potential with SkillChain
+                </h2>
+                <u className='Reinforcement__underline'>
+                </u>
+                <p className='Reinforcement__text'>
```

Comment:
please use double quotes for string properties in JSX. e.g.:  className="Reinforcement__text"

Reply:
Adjusted accordingly!